deployment/puppet/refinery/manifests/python.pp

```diff
@@ -72,12 +80,18 @@
     }
   }
 
-  package { 'virtualenvwrapper': }
+  exec { 'get virtualenvwrapper':
+    command     => "python3 -m pip install virtualenvwrapper",
+    path        => [ '/bin/', '/sbin/' , '/usr/bin/', '/usr/sbin/' ],
+    user        => $app_user,
+    group       => $app_group,
+    require     => Python::Virtualenv[$virtualenv],
+  }
   ->
   file_line { "virtualenvwrapper_config":
-    path    => "/home/${app_user}/.profile",
-    line    => "source /etc/bash_completion.d/virtualenvwrapper",
-    require => Python::Virtualenv[$virtualenv],
+    path        => "/home/${app_user}/.profile",
+    line        => "export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3 && export PROJECT_HOME=/vagrant/refinery && export VIRTUALENVWRAPPER_VIRTUALENV=/usr/bin/virtualenv && export WORKON_HOME=/home/${app_user}/.virtualenv/ && source ./.local/bin/virtualenvwrapper.sh",
```

Comment:
This should suffice:
```shell
VIRTUALENVWRAPPER_PYTHON='/usr/bin/python3'
source /usr/local/bin/virtualenvwrapper.sh
```
https://askubuntu.com/questions/974511/ubuntu-16-04-virtualenvwrapper-python3-6-path

Reply:
we need the `WORKON_HOME` as well but otherwise we got rid of the other fields